versions/1/changelog.yml

```diff
@@ -9,6 +9,9 @@
     link: https://github.com/elastic/package-spec/pull/265
 - version: 1.4.0
   changes:
+  - description: Updates the ml_model file format pattern for validation.
```

Comment:
Please move it to the `1.4.1-next`.

Reply:
Moved in 9b30841d1905134ebfb81f5d24833e6e53446732